### PR TITLE
Add a puppeteer test

### PR DESCRIPTION
**Pull request by PEConn**
I managed to get Jest, Puppeteer and Webpack to play nicely together - it was a surprising amount of effort.

I'm a little concerned that the test takes about 7s for the initial page load, I'll look a bit further into that. For now this can act as a proof of concept.

### CODE REVIEW

**Comment by glennhartmann** [comment]
> it was a surprising amount of effort.

Does not surprise me, tbh.

**Review by glennhartmann — CHANGES_REQUESTED** [review]
Looks good, but I think it needs to be rebased to remove [`ded463f` (#45)](https://github.com/GoogleChromeLabs/llaminator/pull/45/commits/ded463fd86d0eaab9444cc31910d310c8065afe3)

File: jest-puppeteer.config.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (beverloo) [inline jest-puppeteer.config.js:1]: nit: copyright headers here and in `puppeteer.test.js`

File: tests/puppeteer.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// The initial page loads takes ~7 seconds on my machine.
```
Comment (beverloo) [inline tests/puppeteer.test.js:1]: Is that just the first page load, or are we looking at an execution time of ~7 seconds for every test that's being run?

File: tests/puppeteer.test.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+describe('Homepage', () => {
+  beforeAll(async () => {
+    await page.goto('http://localhost:8888', { 'waitUntil': 'domcontentloaded' });
+  })
+
+  it('say Llaminator', async () => {
+    // Note: There is a bug with puppeteer/jest-puppeteer that will cause this test to
+    // fail if run with puppeteer 13.
+    // See https://github.com/smooth-code/jest-puppeteer/issues/461.
```
Comment (beverloo) [inline tests/puppeteer.test.js:12]: That's incredible. Thank you for figuring that out, must've been a pain!

**Review by beverloo — COMMENTED** [review]
Cool, thanks :)

File: tests/puppeteer.test.js (hunk 2 above)
Comment (PEConn) [reply tests/puppeteer.test.js:1]: It's the time taken for the server to start up - the tests themselves are a few milliseconds.

**Comment by PEConn** [comment]
Is this PR good to land? If so I'll merge in the package.json changes then land it.

**Review by beverloo — APPROVED** [review]

**Review by glennhartmann — APPROVED** [review]

**Pull request merged**